EU4toV2/Source/Configuration.cpp

```diff
@@ -109,6 +117,11 @@ void Configuration::instantiate(std::istream& theStream, bool (*DoesFolderExist)
 	parseStream(theStream);
 	clearRegisteredKeywords();
 	setOutputName();
+	if (isHpmEnabled())
+	{
+		Vic2Path = Vic2ModPath + "/HPM";
```

Comment:
You need tests for this behavior.

Reply:
Would it suffice to have two tests: if it is enabled `C:\Vic2ModPath\HPM`, else `C:\Vic2Path`?